### ISSUE TITLE
App Crashes when trying to pick images (java.lang.NullPointerException: tvFolderName must not be null)

### ISSUE THREAD

**Issue by sam-androiddev** [Mindinventory/Lassi-Android#50]
When I try to choose an image from the gallery using LassiOption.CAMERA_AND_GALLERY or LassiOption.GALLERY my app crashes with the following error:

   ```
 java.lang.NullPointerException: tvFolderName must not be null
        at com.lassi.presentation.mediadirectory.adapter.FolderAdapter$FolderViewHolder.bind(FolderAdapter.kt:44)
        at com.lassi.presentation.mediadirectory.adapter.FolderAdapter.onBindViewHolder(FolderAdapter.kt:37)
        at com.lassi.presentation.mediadirectory.adapter.FolderAdapter.onBindViewHolder(FolderAdapter.kt:16)
        at androidx.recyclerview.widget.RecyclerView$Adapter.onBindViewHolder(RecyclerView.java:7254)
        at androidx.recyclerview.widget.RecyclerView$Adapter.bindViewHolder(RecyclerView.java:7337)
        at androidx.recyclerview.widget.RecyclerView$Recycler.tryBindViewHolderByDeadline(RecyclerView.java:6194)
        at androidx.recyclerview.widget.RecyclerView$Recycler.tryGetViewHolderForPositionByDeadline(RecyclerView.java:6460)
        at androidx.recyclerview.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:6300)
        at androidx.recyclerview.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:6296)
        at androidx.recyclerview.widget.LinearLayoutManager$LayoutState.next(LinearLayoutManager.java:2330)
        at androidx.recyclerview.widget.GridLayoutManager.layoutChunk(GridLayoutManager.java:572)
        at androidx.recyclerview.widget.LinearLayoutManager.fill(LinearLayoutManager.java:1591)
        at androidx.recyclerview.widget.LinearLayoutManager.onLayoutChildren(LinearLayoutManager.java:668)
        at androidx.recyclerview.widget.GridLayoutManager.onLayoutChildren(GridLayoutManager.java:170)
        at androidx.recyclerview.widget.RecyclerView.dispatchLayoutStep2(RecyclerView.java:4309)
        at androidx.recyclerview.widget.RecyclerView.dispatchLayout(RecyclerView.java:4012)
        at androidx.recyclerview.widget.RecyclerView.onLayout(RecyclerView.java:4578)
        at android.view.View.layout(View.java:21973)
        at android.view.ViewGroup.layout(ViewGroup.java:6263)
        at androidx.constraintlayout.widget.ConstraintLayout.onLayout(ConstraintLayout.java:1873)
        at android.view.View.layout(View.java:21973)
        at android.view.ViewGroup.layout(ViewGroup.java:6263)
        at android.widget.FrameLayout.layoutChildren(FrameLayout.java:332)
        at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
        at android.view.View.layout(View.java:21973)
        at android.view.ViewGroup.layout(ViewGroup.java:6263)
        at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1829)
        at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1673)
        at android.widget.LinearLayout.onLayout(LinearLayout.java:1582)
        at android.view.View.layout(View.java:21973)
        at android.view.ViewGroup.layout(ViewGroup.java:6263)
        at android.widget.FrameLayout.layoutChildren(FrameLayout.java:332)
        at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
        at android.view.View.layout(View.java:21973)
        at android.view.ViewGroup.layout(ViewGroup.java:6263)
        at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1829)
        at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1673)
        at android.widget.LinearLayout.onLayout(LinearLayout.java:1582)
        at android.view.View.layout(View.java:21973)
        at android.view.ViewGroup.layout(ViewGroup.java:6263)
        at android.widget.FrameLayout.layoutChildren(FrameLayout.java:332)
        at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
        at android.view.View.layout(View.java:21973)
        at android.view.ViewGroup.layout(ViewGroup.java:6263)
        at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1829)
        at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1673)
        at android.widget.LinearLayout.onLayout(LinearLayout.java:1582)
        at android.view.View.layout(View.java:21973)
        at android.view.ViewGroup.layout(ViewGroup.java:6263)
        at android.widget.FrameLayout.layoutChildren(FrameLayout.java:332)
        at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
        at com.android.internal.policy.DecorView.onLayout(DecorView.java:799)
        at android.view.View.layout(View.java:21973)
        at android.view.ViewGroup.layout(ViewGroup.java:6263)
E/AndroidRuntime:     at android.view.ViewRootImpl.performLayout(ViewRootImpl.java:3165)
        at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2675)
        at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1800)
        at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:7751)
        at android.view.Choreographer$CallbackRecord.run(Choreographer.java:967)
        at android.view.Choreographer.doCallbacks(Choreographer.java:791)
        at android.view.Choreographer.doFrame(Choreographer.java:726)
        at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:952)
        at android.os.Handler.handleCallback(Handler.java:883)
        at android.os.Handler.dispatchMessage(Handler.java:100)
        at android.os.Looper.loop(Looper.java:241)
        at android.app.ActivityThread.main(ActivityThread.java:7604)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:941)

```
**I am using following code:** 

```
Intent intent = new Lassi(getActivity()).with(LassiOption.CAMERA_AND_GALLERY)
                .setMaxCount(1)
                .setMediaType(MediaType.IMAGE)
                .setCropType(CropImageView.CropShape.RECTANGLE)
                .setCropAspectRatio(1, 1)
                .enableFlip()
                .enableRotate()
                .enableActualCircleCrop()
                .build();
        imagePickerActivityResultLauncher.launch(intent);
```

**Comment by sanjay-mi** [comment]
Hello @sam-androiddev ,
Thanks for reporting the issue, our team is looking into it. It would be great for us if you can share the below things to understand the cause properly.

* Device specification like (Manufacturer, OS version, etc)
* Library version
* Screenshot/video with steps to reproduce the issue

**Comment by sanjay-mi** [comment]
Hello @sam-androiddev 

Our team tried to regenerate the issue but it's not regenerated. It would be great for us if you share the more details.
We have released lassi 0.3.0. Please use it and let us know in case still if you are facing the issue.

Thank you
